src/Button.js

```diff
@@ -34,7 +34,8 @@ class AriaMenuButtonButton extends React.Component {
       case 'ArrowDown':
         event.preventDefault();
         if (!ambManager.isOpen) {
-          ambManager.openMenu({ focusMenu: true });
+          ambManager.openMenu();
+          ambManager.focusItem(0);
```

Comment:
Yes, this is done automatically, there no need to focus the item again. I'll remove it.